src/main/java/soconnect/model/todo/Priority.java

```diff
@@ -0,0 +1,82 @@
+package soconnect.model.todo;
+
+import static java.util.Objects.requireNonNull;
+import static soconnect.commons.util.AppUtil.checkArgument;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Represents a {@code Todo}'s priority in the {@code TodoList}.
+ * Can only be one of three values: low, medium, or high.
+ */
+public class Priority {
+
+    public static final String MESSAGE_CONSTRAINTS = "Priority can only be `low`, `medium`, or `high`.";
+
+    public static final String PRIORITY_LOW = "low";
+    public static final String PRIORITY_MEDIUM = "medium";
+    public static final String PRIORITY_HIGH = "high";
+
```

Comment:
I don't think it will work because the `TodoList` is sorted by priority which relies on the `compareTo` method as well as the mapping

Reply:
Ooooh, I've learned something new, thanks 👍 

I think I'll keep it as a class instead of enum since all other fields like `Name`, `Description`, `Phone`, etc are all classes. 
Provides some uniformity when it comes to treatment of fields such as `MESSAGE_CONSTRAINTS` of each class as well as their individual `isValid` methods. 

(i.e. I can expect to use `isValid` to check whether any field is valid, and I can `MESSAGE_CONSTRAINTS` of each field to output the corresponding error message to the user)
Does it make sense? 😆

Reply:
Alright, I think we can keep it as a class.